README.md

```diff
@@ -74,3 +74,15 @@ See the [DataFusion Comet User Guide](https://datafusion.apache.org/comet/user-g
 ## Contributing
 See the [DataFusion Comet Contribution Guide](https://datafusion.apache.org/comet/contributor-guide/contributing.html)
 for information on how to get started contributing to the project.
+
+## Running ScalaTest suites from the CLI
+
+Running single ScalaTest suites from the CLI is possible using the `suites`
+argument, for example if you only want to execute the test cases that contains *valid*
+in their name in `org.apache.comet.CometCastSuite` you can use
+
+```sh
+mvn test -Dsuites="org.apache.comet.CometCastSuite valid" -Dskip.surefire.tests=true
+```
+
+Other options for selecting specific suites are described in the [ScalaTest Maven Plugin documentation](https://www.scalatest.org/user_guide/using_the_scalatest_maven_plugin)
```

Comment:
This is helpful content @edmondop but I think this needs to be in the development guide (part of the contributing guide linked to here in the README), which is published at https://datafusion.apache.org/comet/contributor-guide/development.html

The source is in this repo under `docs/source/contributor-guide`

Reply:
Fixed